### ISSUE TITLE
mssql odbc - otherDb..otherTable

### ISSUE THREAD

**Issue by ScottHuangZL** [go-xorm/xorm#336]
I use mssql db through odbc connection.
It is work when i query same db tables.

Same DB Work:
engine2.ShowSQL = true
err = engine2.Table("TSO_Values").Select("Part_Status_Id as id, Part_Status as name ").Limit(10, 0).Find(&users)

**However not work for below:**
err = engine2.Table("OtherDb..OtherTable").Select("Part_Status_Id as id, Part_Status as name ").Limit(10, 0).Find(&users)

It show error like below
**Invalid object name...**
 SELECT  TOP 10 Part_Status_Id as id, Part_Status as name  FROM "OtherDb..OtherTable"

The reason is I find **"** in above sql, it is wrong, we need below. 

 SELECT  TOP 10 Part_Status_Id as id, Part_Status as name  FROM OtherDb..OtherTable

**Please fix it. I think you can check .. when generate sql, thanks.**

**Comment by lunny** [comment]
Hi, I think maybe you should use only one dot between dbname and table name. Could you change the code to
```Go
err = engine2.Table("OtherDb.OtherTable").Select("Part_Status_Id as id, Part_Status as name ").Limit(10, 0).Find(&users)
```
And try again?

**Comment by ScottHuangZL** [comment]
No, it do not work.
The key issue is mssql do not recognize quotation marks in the sql which generate by xorm.
Wrong: select * from "Otherdb.OtherTable" 
Wrong: select * from "Otherdb..OtherTable"
Correct: select * from Otherdb..OtherTable

So, why xorm add additional " for the table name??
Thanks.

**Comment by lunny** [comment]
Because sometimes maybe your table's name is conflict with the database keyword. So currently xorm will add single qutoe on any table name and column name.

**Comment by lunny** [comment]
You can try
```Go
err = engine2.Table("`OtherDb`..`OtherTable`").Select("Part_Status_Id as id, Part_Status as name ").Limit(10, 0).Find(&users)
```
before I try to fix this issue.

**Comment by ScottHuangZL** [comment]
Thanks for reply, it did not work.
It will generate SQL like : select * from "OtherDB".."OtherTable"

The show SQL flag tell me xorm add double quote instead of single quote.
MSSQL do not accept 'OtherDB..OtherTable' format too.

So, the best way is not add quote for table name.  Or remove quote in case find .. in the table.

Btw, in case some people's table name conflict with database keyword, it still can work, since he/she able create that table in his/her database, no need to worry about. Or that people can use below:
select * from KeyWordTable as NormalWordTable

Please review my proposal, thanks.

**Comment by lunny** [comment]
Yes. I think we can add one function `NoQuote` to stop adding quote automatically.

**Comment by lunny** [comment]
resolved by 15f11df
